docs/workflows/public_data_sharing/retrieve_srr_metadata.md

```diff
@@ -0,0 +1,50 @@
+# Retrieve SRR Metadata Workflow
+
+## Quick Facts
+
+| **Workflow Type** | **Applicable Kingdom** | **Last Known Changes** | **Command-line Compatibility** | **Workflow Level** |
+|---|---|---|---|---|
+| [Public Data Sharing](../../workflows_overview/workflows_type.md/#public-data-sharing) | [Any Taxa](../../workflows_overview/workflows_kingdom.md/#any-taxa) | PHB v2.2.1 | Yes | Sample-level |
```

Comment:
Note that the Last known change is going to be the next release version of PHB, not the current!